scoap3/modules/frontpage/templates/scoap3_frontpage/home.html

```diff
@@ -40,7 +40,7 @@
 
           <div class="tab-content ">
             <div class="tab-pane active" id="1">
-              <h3 class="text-left">Browse by journal:</h3>
+              <h3></h3>
```

Comment:
Do we need the empty h3 ?

Reply:
leaving <h3> empty is just because of styling, not to have 0 gap between tab and list